src/actions/device.ts

```diff
@@ -93,12 +94,44 @@ export const savePresetToRemote = (name: string): AppThunk =>
 		oscQueryBridge.sendPacket(writePacket(message));
 	};
 
+export const loadPatcher = (name: string, inst: number = 0): AppThunk =>
+	() => {
+			let message = {
```

Comment:
whitespace?

Reply:
we use tabs so these show up weird on github, maybe there is some rendering setting we can do for the repo